### DISCUSSION TITLE
Works on Kubuntu 22.04

### DISCUSSION THREAD

**Discussion by pdzaffina** [jdjaxon/linux_cac#18] [General]
Works flawlessly with Chrome and Kubuntu 22.04. I didn't try the Firefox uninstall snap feature. It looks good though. I like how it's built into the shell script installer.

**Comment by jdjaxon** [comment]
Thank you for the feedback! Let me know what you think if you end up trying the uninstall snap portion.

**Comment by riclin85** [comment]
Hi, Im glad i found some people that had a solution for a very similar issue to my issue. Im trying to be able to use my CAC and CAC reader on this laptop that i installed Ubuntu 22.04 onto it. Could you please walk me through the process and the terminal commands so i can make it work? I would really appreciate it.

Im not savy at all with computers, but im pushing my self to learn some before i get too old and too obsolete so i decided to start with Linux OS eventhough I know other OS's are way beginner friendlier but i also believe that Linux will pay off in the long run.

Thank you in advance and have a great weekend.
